docs/usage/PRODUCT_SYNC.md

```diff
@@ -1,4 +1,4 @@
-# commercetools product sync
+# Product Sync
```

Comment:
I didn't want to add links on the titles because:
a. Link to a CTP Product as an entity is different than the Product Sync.
a. Hyperlink titles don't look good.

However, I changed the descriptions and added links to the CTP entities 😉 4ef0c3e